Owl/app/src/main/java/com/example/owl/ui/courses/Courses.kt

```diff
@@ -109,11 +93,21 @@ fun CoursesAppBar() {
     }
 }
 
-private enum class CourseTabs(
+enum class CourseTabs(
     @StringRes val title: Int,
-    @DrawableRes val icon: Int
+    @DrawableRes val icon: Int,
+    val route: String
 ) {
-    MY_COURSES(R.string.my_courses, R.drawable.ic_grain),
-    FEATURED(R.string.featured, R.drawable.ic_featured),
-    SEARCH(R.string.search, R.drawable.ic_search)
+    MY_COURSES(R.string.my_courses, R.drawable.ic_grain, CoursesDestinations.MY_COURSES_ROUTE),
+    FEATURED(R.string.featured, R.drawable.ic_featured, CoursesDestinations.FEATURED_ROUTE),
+    SEARCH(R.string.search, R.drawable.ic_search, CoursesDestinations.SEARCH_COURSES_ROUTE)
+}
+
+/**
+ * Destinations used in the ([OwlApp]).
+ */
+private object CoursesDestinations {
```

Comment:
Could we use inline classes to make this type safe?

Reply:
navigation-compose uses strings, can you expand on what you mean?

Reply:
I haven't played with inline classes but wondered if they could give us type safety i.e. define a `CourseDestination`type that becomes a raw string at runtime and kept Nav happy. Again, haven't tried it, just dislike stringly typeing.

Reply:
It wouldn't add any benefit, as we can't tell navigation to use our class instead of String.

There's more safety coming so let's keep it stringly for now.